Las/Utils/Authorization.cs

```diff
@@ -0,0 +1,194 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+using Lucidtech.Las.Core;
+
+namespace Lucidtech.Las.Utils
+{
+    public class AmazonAuthorization
+    {
+        /* General information */
```

Comment:
Looks like comment convention is inconsitent. /* */ here and /// or // elsewhere. Is this intended?

Reply:
No, will remove comment completely since it does not really contribute to anything.